girder_worker/app.py

```diff
@@ -4,6 +4,7 @@
 from distutils.version import LooseVersion
 from celery.signals import (task_prerun, task_postrun,
                             task_failure, task_success, worker_ready)
+from girder_client import GirderClient
```

Comment:
girder-client currently isn't a hard requirement of Girder Worker (unless using the girder_io plugin). This PR would require it become one.